### ISSUE TITLE
Please make sort confirmation optional

### ISSUE THREAD

**Issue by ziffusion** [heftig/sort-bookmarks#18]
Most people just want to sort their bookmarks with the configuration they choose once. Having to click the confirmation is a unnecessary inconvenience for such flows.

If people want to pause and change settings for each sort, maybe there can be a binding that triggers that flow e.g. shift+mouse.

**Comment by heftig** [comment]
I'm not sure what you're getting at. You want to save a single click?

**Comment by ziffusion** [comment]
Yes. So by default, when I pick the option to "Sort this folder" in the drop down menu, it should just sort it without popping up the window asking for confirmation.

And if you want to retain 2 flows (with and without confirmation), maybe 2 different key bindings can initiate each flow e.g. simple click may trigger the default flow and shift-click may trigger the other flow.

**Comment by heftig** [comment]
Sorting a single folder actually saves folder-specific sort settings so the confirmation is not eliminated so easily.

**Comment by ziffusion** [comment]
Just curious. Why not?

In the flow which does not ask for confirmation, it can just use the settings that were saved last. Or defaults. I mean, the 2 flows could like so:

1. Default flow : key binding (click) : Sort the folder using the previously saved preferences. If no preferences have been saved then either use defaults, or prompt for preferences.
2. Alternative flow :  key binding (shift+click) : Prompt for preferences and then sort.
